index.html

```diff
@@ -20,35 +20,38 @@
 <body data-theme="theme1">
 
   <nav class="top">
-    <a id="toplink" href="#footerlink">Ugrás a lap aljára</a>
+    <a id="toplink" href="#footerlink" aria-label="Jump to the bottom of the page">Ugrás a lap aljára</a>
   </nav>
 
-  <div class="base-image">
+  <div class="base-image" role="figure">
     <img src="https://picsum.photos/2200/400" alt="Lorem ipsum dolor sit amet, consectetur adipiscing elit"
-         loading="lazy">
+         loading="lazy" >
   </div>
   <nav class="sublinks">
-    <a href="#subsipsum1">Lorem subIpsum 1</a>
-    <a href="#subsipsum2">Lorem subIpsum 2</a>
-    <a href="#subsipsum3">Lorem subIpsum 3</a>
-    <a href="#subsipsum4">Lorem subIpsum 4</a>
-    <a href="#subsipsum5" target="_blank">Lorem subIpsum 5</a>
-    <a href="#subsipsum6" target="_blank">Lorem subIpsum 6</a>
+    <a href="#subsipsum1" aria-label="Jump to article paragraph" >Lorem subIpsum 1</a>
+    <a href="#subsipsum2" aria-label="Jump to article paragraph" >Lorem subIpsum 2</a>
+    <a href="#subsipsum3" aria-label="Jump to article paragraph" >Lorem subIpsum 3</a>
+    <a href="#subsipsum4" aria-label="Jump to article paragraph" >Lorem subIpsum 4</a>
+    <a href="#subsipsum5" aria-label="Jump to article paragraph" target="_blank">Lorem subIpsum 5</a>
+    <a href="#subsipsum6" aria-label="Jump to article paragraph" target="_blank">Lorem subIpsum 6</a>
```

Comment:
Itt is konkretizalni, hogy melyik link melyik paragrafushoz ugrik pontosan (1-es, 2-es, stb)

Reply:
Köszönöm szépen a code review-t, megnézem majd a javasolt módosításokat!